src/aurelia-binding.d.ts

```diff
@@ -509,6 +512,28 @@ export declare interface InternalCollectionObserver {
    * @param callable A callable object.
    */
   unsubscribe(context: any, callable: Callable): void;
+  /**
+   * Adds a change record to the collection observer.
+   * @param changeRecord 
+   */
+  addChangeRecord(changeRecord: ICollectionObserverSplice): void;
+  /**
+   * This will flush the change records of this observer and call any subscribers if applicable.
+   */
+  flushChangeRecords(): void;
```

Comment:
I guess this can be exposed since it serves a similar purpose to `flushMicroTaskQueue`, and I can see potential legitimate use cases for this.

Reply:
Awesome!